website/pages/api-docs/secret/kmip/index.mdx

```diff
@@ -428,6 +428,66 @@ $ curl \
 }
 ```
 
+## Sign CSR
+
+| Method | Path                                                |
+| :----- | :-------------------------------------------------- |
+| `POST` | `/kmip/scope/:scope/role/:role/credential/sign` |
+
+Create a new client certificate tied to the given role and scope,
+based on a Certificate Signing Request (CSR) provided as input.
+The key type and key bits used in the CSR must match those of the role.
+
+### Parameters
+
+- `scope` (`string: <required>`) - Name of scope. This is part of the request URL.
+- `role` (`string: <required>`) - Name of role. This is part of the request URL.
+- `format` (`string: "pem"`) - Format to return the certificate, private key,
+  and CA chain in. One of `pem`, `pem_bundle`, or `der`.
+- `csr` (`string`) - CSR in PEM format.
+
+### Sample Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data '{"csr": "-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
Can the JSON value be provided like so, or do the newlines need to be `\n`?

Reply:
Good catch, you're right they must be `\n`.